### DISCUSSION TITLE
Combine people and albums

### DISCUSSION THREAD

**Discussion by pelinoleg** [damongolding/immich-kiosk#28] [Q&A] [answered]
Is there a way to show photos from both faces and albums?

**Comment by damongolding** [comment, marked as answer]
Not currently @pelinoleg . I did try to implement this when I first started Kiosk but due to a limitation in the Immich API I was unable to implement that feature in a way that I was happy with.

**Comment by pelinoleg** [comment]
ok. I hope this will appear in the future. Thanks

Can I write more suggestions for adding functionality to the immi-kiosk?

**Reply by damongolding** [reply]
By all means, please do!

Just add them to [the ideas discussion](https://github.com/damongolding/immich-kiosk/discussions/categories/ideas).

I'll keep an eye on the API and if that changes or I come with a better way to implement people in album I will update this.

**Reply by drenkmann** [reply]
As far as I understand, albums and people are managed as seperate "buckets" (that's what the docs call them at least). Would randomly selecting a bucket be a solution to combining multiple sources? That would bypass the requirement for the immich api to allow something like this, right?

I don't know how pictures are currently selected so maybe that's just not how it works, but for me at least randomly switching between buckets would be a satisfactory solution. Maybe others disagree though.

PS: I'm not sure if the original suggestion meant showing photos of specific people that are in a specific album (AND) or showing a picture from either an album or a person (OR)? I'm referring to the latter.

**Reply by damongolding** [reply]
This request was asking if it was possible to show specific people within a specific album @drenkmann. The latter is how Kiosk currently works.

**Reply by drenkmann** [reply]
@damongolding Oh I'm sorry then. The people documentation said that people can't be combined with albums, so I assumed that it didn't work at all. Sorry for the misunderstanding.

**Reply by damongolding** [reply]
> @damongolding Oh I'm sorry then. The people documentation said that people can't be combined with albums, so I assumed that it didn't work at all. Sorry for the misunderstanding.

Are you referring the to `require_all_people` feature docs? If so that's to try and clarify that you can't limit/filter other sources (buckets) by people.

**Reply by drenkmann** [reply]
> Are you referring the to `require_all_people` feature docs? If so that's to try and clarify that you can't limit/filter other sources (buckets) by people.

Yes, I am. I thought that note was referring to the entire people section, not just require_all_people, that's why I got confused. Sorry for the late reply.
